src/main/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocument.java

```diff
@@ -698,157 +735,177 @@ public boolean generateDocumentGeneralLedgerPendingEntries(GeneralLedgerPendingE
-         * KFSPTS-764
-         * Returns true when the wire charge amount is found to be zero dollars based on the bank country code
-         * otherwise, returns false.
-         *
-         * @param wireCharge
-         * @return true when wire charge for DV bank is zero dollars.
-         */
-       private boolean isZeroDollarWireCharge(WireCharge wireCharge) {
+    /**
+     * KFSPTS-764
+     * Returns true when the wire charge amount is found to be zero dollars based on the bank country code
+     * otherwise, returns false.
+     *
+     * @param wireCharge
+     * @return true when wire charge for DV bank is zero dollars.
+     */
+   private boolean isZeroDollarWireCharge(WireCharge wireCharge) {
 
-            if ( (KFSConstants.COUNTRY_CODE_UNITED_STATES.equals(getWireTransfer().getBankCountryCode()) && wireCharge.getDomesticChargeAmt().isZero()) ||
-                 (!KFSConstants.COUNTRY_CODE_UNITED_STATES.equals(getWireTransfer().getBankCountryCode()) && wireCharge.getForeignChargeAmt().isZero()) ){
-                //DV is for a US bank and wire charge value is zero dollars OR DV is for a foreign bank and wire charge is zero dollars.
-                return true;
-            }
-            return false;
+        if ( (KFSConstants.COUNTRY_CODE_UNITED_STATES.equals(getWireTransfer().getBankCountryCode()) && wireCharge.getDomesticChargeAmt().isZero()) ||
+             (!KFSConstants.COUNTRY_CODE_UNITED_STATES.equals(getWireTransfer().getBankCountryCode()) && wireCharge.getForeignChargeAmt().isZero()) ){
+            //DV is for a US bank and wire charge value is zero dollars OR DV is for a foreign bank and wire charge is zero dollars.
+            return true;
         }
+        return false;
+    }
 
-        @Override
-        public boolean answerSplitNodeQuestion(String nodeName) throws UnsupportedOperationException {
-            if (nodeName.equals(CuDisbursementVoucherDocument.DOCUMENT_REQUIRES_AWARD_REVIEW_SPLIT))
-                return isCAndGReviewRequired();
-            if (nodeName.equals(CuDisbursementVoucherDocument.DOCUMENT_REQUIRES_CAMPUS_REVIEW_SPLIT))
-                return isCampusReviewRequired();
-            if (nodeName.equals(DisbursementVoucherDocument.DOCUMENT_REQUIRES_TAX_REVIEW_SPLIT)) {
-                return isTaxReviewRequired();
-            }
-            if (nodeName.equals(DisbursementVoucherDocument.DOCUMENT_REQUIRES_TRAVEL_REVIEW_SPLIT)) {
-                return isTravelReviewRequired();
-            }
-            if (nodeName.equals(DOCUMENT_REQUIRES_SEPARATION_OF_DUTIES)) {
-                return isSeparationOfDutiesReviewRequired();
-            }
-            throw new UnsupportedOperationException("Cannot answer split question for this node you call \""+nodeName+"\"");
+    @Override
+    public boolean answerSplitNodeQuestion(String nodeName) throws UnsupportedOperationException {
+        if (nodeName.equals(CuDisbursementVoucherDocument.DOCUMENT_REQUIRES_AWARD_REVIEW_SPLIT))
```

Comment:
Yeah, that would probably be a good refactoring to avoid NPEs, but I just reformatted this part of the code, I wasn't really reviewing it for possible changes.